arangod/Agency/Supervision.cpp

```diff
@@ -1644,6 +1648,89 @@ void Supervision::cleanupFinishedAndFailedJobs() {
   cleanup(failedPrefix, maximalFailedJobs);
 }
 
+// Guarded by caller
+void arangodb::consensus::cleanupHotbackupTransferJobsFunctional(
+    Node const& snapshot,
+    std::shared_ptr<VPackBuilder> envelope) {
+  // This deletes old Hotbackup transfer jobs in 
+  // /Target/HotBackup/TransferJobs according to their time stamp.
+  // We keep at most 100 transfer jobs which are completed.
+  constexpr uint64_t maximalNumberTransferJobs = 100;
+  constexpr char const* prefix = "/Target/HotBackup/TransferJobs/";
+
+  auto const& jobs = snapshot.hasAsChildren(prefix).first;
+  if (jobs.size() <= maximalNumberTransferJobs + 6) {
```

Comment:
The +6 looks like a magic number here. What is its purpose? A code comment would help!

Reply:
Done.